config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml

```diff
@@ -25,7 +25,7 @@ presubmits:
         - --
         - --build=bazel
         - --deployment=local
-        - --ginkgo-parallel=1
+        - --ginkgo-parallel
```

Comment:
this definitely makes things slightly flakier for `kind` at least.
https://testgrid.k8s.io/conformance-kind
https://testgrid.k8s.io/conformance-kind#kind,%20master%20(dev)%20%5Bnon-serial%5D

Reply:
I plan to experiment with values for this at some point to see if reducing it helps, otoh to me this signals that our tests are bad, I'm pretty certain that we have some conformance tests using events which I thought was a no-no ...